### DISCUSSION TITLE
NOTICE: Please describe your question in English

### DISCUSSION THREAD

**Discussion by davidzollo** [apache/dolphinscheduler#7255] [General]
As a global project, please start a new discussion in English, thanks

**Comment by wangchuang07** [comment]
I am using version 3.2.1, and the number of front-end interface lists has exceeded the limit of this page, but it has not been paginated. For example, there are a total of 43 workflows on the interface, with 10 displayed on each page but only 1 page. Is there a configuration issue?
<img width="434" alt="bugpage" src="https://github.com/apache/dolphinscheduler/assets/38686005/350461f5-c56c-4072-ac50-d35f3012b412">

**Reply by edclol** [reply]
I also encountered a problem with the backend code
